builds/misc/images-release.yaml

```diff
@@ -386,11 +386,9 @@ jobs:
 ################################################################################
     displayName: Windows
     pool:
-      name: Azure-IoT-Edge-Core
+      name: $(pool.windows.name)
       demands: 
-        - Build-Image -equals true
-        - win-rs5
-        - DotNetFramework
+       - ImageOverride -equals agent-aziotedge-winserver-2019dc-build
```

Comment:
Are we using Windows Server for the Windows? I thought we should be using Windows Enterprise LTS for this. @damonbarry